viewer/static/js/app/components/views/chart.js

```diff
@@ -88,13 +88,18 @@ export default class extends React.Component {
                 axisType="x"
                 refLabels={props.refLabels}
                 size={props.size.innerHeight}
+                height={props.size.height}
+                width={props.size.width}
+                xunit={props.xunit}
               />
               <ChartAxisContainer
                 metricId={props.metricId}
                 scale={props.yScale}
                 axisType="y"
                 refLabels={props.refLabels}
                 size={props.size.innerWidth}
```

Comment:
We use 'size' to mean width (within the child component) in some cases and possibly other things (like the size object containing DOM dimensions) later depending on context. We could send all of the size object or just rename things better. Maybe I shouldn't have mentioned it as it's not specific to this commit but it seems weird now when I see the project from the outside.